pages/DSNP/DSNP-Messages.md

```diff
@@ -62,19 +62,27 @@ a public post (was Announcement)
 
 | field     | description | type |
 |-------    |-------------| ----|
-| inReplyTo | messageID replied to | bytes32
+| dnspRoute (fromAddress) | id of the sender | bytes20
```

Comment:
I thought the idea here was to remove this field from the DSNP data so it wouldn't be duplicated.

Reply:
It is. I don't like how I am indicating it. The idea is to note what data the dsnpRoute contains, but you are right that it doesn't look that way. I'll work on a better way to show that information.

Reply:
Here are three different ways to show the data. Not sure which I like:

<img width="735" alt="image" src="https://user-images.githubusercontent.com/1252199/109030387-c016fe80-7691-11eb-894a-a7671b616dd0.png">

Reply:
@shannonwells Updated to a new style